lms/envs/production.py

```diff
@@ -1127,3 +1127,7 @@
 ######################### Settings for LUMS #######################
 
 GOOGLE_ANALYTICS_ID = AUTH_TOKENS.get('GOOGLE_ANALYTICS_ID', GOOGLE_ANALYTICS_ID)
+
+################################ CONTACT US SUPPORT EMAILS #####################
+
+CONTACT_US_SUPPORT_EMAILS = ENV_TOKENS.get('CONTACT_US_SUPPORT_EMAILS', [])
```

Comment:
Please create the same `CONTACT_US_SUPPORT_EMAILS` variable in common.py with default value as `[ ]` and use here as a default value.